mlx4: Change release date to 2008, and version to 1.0 (since is GA).

Signed-off-by: Jack Morgenstein <jackm@dev.mellanox.co.il>

---
 drivers/net/mlx4/mlx4.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: ofed_kernel/drivers/net/mlx4/mlx4.h
===================================================================
--- ofed_kernel.orig/drivers/net/mlx4/mlx4.h
+++ ofed_kernel/drivers/net/mlx4/mlx4.h
@@ -47,8 +47,8 @@
 
 #define DRV_NAME	"mlx4_core"
 #define PFX		DRV_NAME ": "
-#define DRV_VERSION	"0.01"
-#define DRV_RELDATE	"May 1, 2007"
+#define DRV_VERSION	"1.0"
+#define DRV_RELDATE	"April 4, 2008"
 
 enum {
 	MLX4_HCR_BASE		= 0x80680,
